cython/gtsam/examples/SFMExample.py

```diff
@@ -20,7 +20,7 @@
                          GenericProjectionFactorCal3_S2, Marginals,
                          NonlinearFactorGraph, PinholeCameraCal3_S2, Point3,
                          Pose3, PriorFactorPoint3, PriorFactorPose3, Rot3,
-                         SimpleCamera, Values)
+                         PinholeCameraCal3_S2, Values)
```

Comment:
We still have a cython folder?

Reply:
I didn't delete it to limit the number of diffs, will delete after review :)